NEWS.txt

```diff
@@ -71,6 +71,24 @@ using the provided 'sstableupgrade' tool.
 
 New features
 ------------
+    - JDK21 and Generational ZGC is now officially supported. By default, when JDK21 is used the server params in
```

Comment:
That sounds great, do we recommend this in prod already? I don't think I am the right person to review the selected settings and say they are good for prod.